api/scripts/update-certification-challenges-with-skill-id.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+const _ = require('lodash');
+const Promise = require('bluebird');
```

Comment:
On évite d'écraser l'objet `Promise` natif (contrairement à la suggestion de la documentation `bluebird`) :

```suggestion
const bluebird = require('bluebird');
```

Reply:
Ok